sdk/basyx/objectstore.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2023 the Eclipse BaSyx Authors
+#
+# This program and the accompanying materials are made available under the terms of the MIT License, available in
+# the LICENSE file of this project.
+#
+# SPDX-License-Identifier: MIT
+"""
+This module implements Registries for the AAS, in order to enable resolving global
+:class:`Identifiers <basyx.aas.model.base.Identifier>`; and mapping
+:class:`Identifiers <basyx.aas.model.base.Identifier>` to :class:`~basyx.aas.model.base.Identifiable` objects.
+"""
+
+import abc
+from typing import MutableSet, Iterator, Generic, TypeVar, Dict, List, Optional, Iterable
+
+from aas_core3.types import Identifiable, Referable, Class
+
+Identifier = str
+Id_short = str
```

Comment:
Do we need to define those two here, as in, do these not exist in `aas_core3.types`? 
A much harder question: Do we actually want to define these types or use `str` itself as input in our methods? Could you check how `aas_core3` does that in their class `__init__` functions? If we do it differently, it might clash with their `verification.py` functionality. 

Is that the way `Id_short` is written in aas-core?

Reply:
As far as I can see aas_core3 uses no additional abstraction layer regarding the indentifier and id_short type. Both are just defined as `str`
### identifier
The Identifiable class specifies the 'globally unique identifier' as a plain string.
```
class Identifiable(Referable):
    [...]
    id: str
```

### id_short
E.g. the Referable class just uses
```
class Referable(HasExtensions):
    [...]
    id_short: Optional[str]
```
So we should be fine by just using `str` instead of an additional self defined type.
Also it is written `id_short` everywhere in the aas_core3 project, I could not find any matches to `Id_short`.

Reply:
Should be fixed now

Reply:
We decided to not introduce these two alias types, since that would not fit with the `verification`-pattern implemented in aas-core. 
Therefore, we chose to use the `str` type wherever these types would occur.